rust/src/bittorrent_dht/bittorrent_dht.rs

```diff
@@ -0,0 +1,401 @@
+/* Copyright (C) 2020 Open Information Security Foundation
```

Comment:
nit 2021

Reply:
Just to confirm, this is only for new files and the copyright year of old modified files should not be updated to 2021 as well?

Reply:
New files should have 2021
Modified files should have startyear-2021